geode-management/src/main/java/org/apache/geode/management/internal/JavaClientClusterManagementFactory.java

```diff
@@ -13,22 +13,21 @@
  * the License.
  */
 
-package org.apache.geode.management.spi;
+package org.apache.geode.management.internal;
 
 import org.springframework.http.client.ClientHttpRequestFactory;
 
 import org.apache.geode.management.api.ClusterManagementService;
 import org.apache.geode.management.client.ClusterManagementServiceProvider;
-import org.apache.geode.management.internal.ClusterManagementClient;
 
 /**
  * A Service Provider factory which will be used to create {@link ClusterManagementService}
  * instances from pure Java code - i.e. the Cluster Management endpoint URL <b>cannot</b> be
  * inferred from the implied runtime context but needs to be specifically configured using a given
  * URL or {@code ClientHttpRequestFactory}.
  */
-public class JavaClientClusterManagementProviderFactory
-    implements ClusterManagementServiceProviderFactory {
+public class JavaClientClusterManagementFactory
```

Comment:
For consistency I suggest that this be renamed `JavaClientClusterManagementServiceFactory`